src/app/ui/timeline.cpp

```diff
@@ -1518,6 +1538,162 @@ void Timeline::drawCel(ui::Graphics* g, LayerIndex layerIndex, frame_t frame, Ce
   // Draw decorators to link the activeCel with its links.
   if (data->activeIt != data->end)
     drawCelLinkDecorators(g, bounds, cel, frame, is_active, is_hover, data);
+
+
+  if (docPref().celPreview.showThumb() && image) {
+
+    she::Surface *thumb_surf;
+
+    // TODO select alphas and color of background
+    // TODO scale alternatives (bilinear, ...)
+
+    gfx::Rect thumb_bounds = gfx::Rect(bounds).offset(1,1).inflate(-1,-1);
+
+    double zw = thumb_bounds.w / (double)image->width();
+    double zh = thumb_bounds.h / (double)image->height();
+    double z0 = zw < zh ? zw : zh;
+    double zoom = z0 > 1 ? 1 : z0;
```

Comment:
I think it would be nice to enclose these `?` expressions with a `(` and `)`. E.g.

    double z0 = (zw < zh ? zw : zh);